versioned_docs/version-v2.16.x/extend/extend-apiml/api-mediation-oidc-authentication.md

```diff
@@ -167,6 +170,16 @@ For more information about the Zowe CLI Identity Federation Plug-in, see the [RE
 - **`components.gateway.apiml.security.oidc.registry`**  
    Specifies the SAF registry used to group the identities recognized as having a OIDC identity mapping. The registry name is the string used during the creation of the mapping between the dustributed and mainframe user identities. For more information, see the [ESM configuration](#esm-configuration).
 
+- **`components.gateway.apiml.security.oidc.validationType`**  
+   Specifies the validation type for OIDC authentication functionality, which is set to `JWK` by default. To enable access token validation using a remote endpoint set this property to `endpoint`. When set to `endpoint`, the Gateway uses the URI sepecified in the property `userInfo` to validate access tokens.
```

Comment:
Please add a comma after "To enable access token validation using a remote endpoint"